components/sync15/src/client.rs

```diff
@@ -222,7 +222,12 @@ impl Sync15StorageClient {
     where
         for<'a> T: serde::de::Deserialize<'a>,
     {
-        log::trace!("request: {} {}", req.method, req.url.path());
+        log::trace!(
```

Comment:
Logging the options obviously isn't necessary for this patch, but it does make verifying things are working as expected easier.